.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!-- AICA Pull Request Template: 10 easy steps for a successful PR!
+1. Give the PR a relevant and descriptive title
+2. Link the PR to the parent issue, which should already describe and motivate the PR
+3. Explain how the PR addresses the parent issue
+6. Add any supporting resources (screenshots, test results, etc)
+7. Help the reviewer by suggestion the method and estimated time to review
+8. If applicable, make a checklist of tasks that should be completed before merging
+9.  If applicable, link related issues (blocked by / blocks)
+10. Tag reviewers
```

Comment:
Do you think this section is useful, or is it just duplicating the information in the body below?

Also I realized there is a step missing. To officially link the PR to the parent issue, it needs to be done under the "Development" field on the right hand side of the page. So, maybe this can be simplified to just:
```
- Give the PR a relevant and descriptive title
- Update the description following the template below
- Tag relevant reviewers under "Reviewers"
- Create the PR
- Link the PR with its parent issue under "Development"
```

Reply:
I do believe it useful not because it's duplicating the information below but it is a constant reminder of what the good practices are.

Maybe the 2nd point can be more precise saying that you should link the issue on the right side but I liked the more verbose version.